src/uptane/tuf.cc

```diff
@@ -91,12 +91,9 @@ bool Hash::MatchWith(const std::string &content) const {
 
 bool Hash::operator==(const Hash &other) const { return type_ == other.type_ && hash_ == other.hash_; }
 
-Target::Target(const std::string &filename, const Json::Value &content) : filename_(filename), ecu_identifier_("") {
+Target::Target(const std::string &filename, const Json::Value &content) : filename_(filename) {
   if (content.isMember("custom")) {
-    Json::Value custom = content["custom"];
-    if (custom.isMember("ecuIdentifier")) {
-      ecu_identifier_ = content["custom"]["ecuIdentifier"].asString();
-    }
+    custom_ = content["custom"];
```

Comment:
It is because image and director has different formats of custom field, for example director contains "ecuIdentifier" but image not. But image contains hardware identifiers which might be needed in the future.